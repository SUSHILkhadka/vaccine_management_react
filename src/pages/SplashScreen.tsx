import { Spin } from 'antd';
import React from 'react';
import '../styles/Page.scss'
const SplashScreen: React.FC = () => {
  return (
    <div className='page--splash'>
      <Spin size='large' />
    </div>
  );
};
export default SplashScreen;
