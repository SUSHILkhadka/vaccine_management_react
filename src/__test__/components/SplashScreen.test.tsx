import "@testing-library/jest-dom"; //important to toBeInDocument() to run
import SplashScreen from "../../pages/SplashScreen";
import { render } from "@testing-library/react";
import React from "react"

it("should render splash screen properly", () => {
  const page = render(<SplashScreen />);
  console.log(page.asFragment())
  expect(page.asFragment()).toMatchSnapshot();
});
