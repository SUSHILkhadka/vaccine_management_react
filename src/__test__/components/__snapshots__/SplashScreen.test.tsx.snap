// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render splash screen properly 1`] = `
<DocumentFragment>
  <div
    class="page--splash"
  >
    <div
      aria-busy="true"
      aria-live="polite"
      class="ant-spin ant-spin-lg ant-spin-spinning"
    >
      <span
        class="ant-spin-dot ant-spin-dot-spin"
      >
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
      </span>
    </div>
  </div>
</DocumentFragment>
`;
